projects/plugins/jetpack/modules/search/instant-search/components/search-app.jsx

```diff
@@ -59,9 +59,13 @@ class SearchApp extends Component {
 	}
 
 	componentDidMount() {
+		// Do not flush this debounced invocation.
```

Comment:
I think we can probably drop the first line of the comment here, because it makes reference to code that has been removed.

Reply:
I wonder if the sequence should be added when sending out requests rather than already when they are responded? Because when the response is back, they've already have their natural order no matter we add the sequence or not.